compiler/resolution/virtualDispatch.cpp

```diff
@@ -285,7 +285,14 @@ static void collectMethods(FnSymbol*               pfn,
           }
         }
         if (possibleSignatureMatch(pfn, cfn) == true) {
-          methods.push_back(cfn);
+          if (cfn->retTag == RET_PARAM ||
+              cfn->retTag == RET_TYPE) {
+            USR_FATAL_CONT(cfn,
+                           "param default arguments in overriden classes "
```

Comment:
"overridden classes" or "methods"?  In either case, note that overridden is misspelled (right?  I'm terrible at double letters as @cassella will attest).